### PR TITLE
table: html row automerge support

### PR DESCRIPTION
**Pull request by hindessm**
## Proposed Changes

Support AutoMerge across row entries in HTML rendering mode.

Thanks for a great library.

### CODE REVIEW

File: table/render_html_test.go [hunk 1]
```diff
@@ -517,3 +517,29 @@ func TestTable_RenderHTML_Sorted(t *testing.T) {
...
+  <tr>
+    <td align="center" colspan=2>Y</td>
+  </tr>
+  <tr>
+    <td>Y</td>
+    <td>N</td>
+  </tr>
+  </tbody>
+</table>`)
+}
```
Comment (jedib0t) [inline table/render_html_test.go]: Can you add a test with a more complex use-case... with a couple of columns without any merge in the same row where a couple of columns are merged? Check the automerge test file for ideas?

**Comment by jedib0t** [comment]
I really appreciate your contribution. If you can add one more test with the above use-case, it will help assure its functionality for more than a simple use-case. Thanks!

File: table/render_html_test.go (hunk 1 above)
Comment (beanz) [reply table/render_html_test.go]: Sure. I've forced pushed with an extra test that hopefully covers enough alternative merge/non-merge options.

**Review by jedib0t — APPROVED** [review]

**Comment by beanz** [comment]
Thanks!

**Comment by hindessm** [comment]
Thanks.

**Comment by jedib0t** [comment]
Hey @hindessm ... I've cut a new tag for you: https://github.com/jedib0t/go-pretty/releases/tag/v6.5.9

**Comment by hindessm** [comment]
That's very kind of you. Thanks.

**Pull request merged**